test/relay_client_test.js

```diff
@@ -450,26 +451,49 @@ contract('RelayClient', function (accounts) {
     })
 
 
-    it("should send relay balance to owner after removed", async function () {
-
-        let response = await request(localhostOne+'/getaddr');
-        let relayServerAddress = JSON.parse(response.body).RelayServerAddress;
-        let beforeOwnerBalance = await web3.eth.getBalance(relayOwner);
-        let res = await rhub.removeRelayByOwner(relayServerAddress, {from:relayOwner});
-        assert.equal("RelayRemoved", res.logs[0].event);
-        assert.equal(relayServerAddress.toLowerCase(), res.logs[0].args.relay.toLowerCase());
-
-        let i = 0;
-        let relayBalance = await web3.eth.getBalance(relayServerAddress);
-        while (relayBalance != 0 && i < 20) {
-            await testutils.sleep(200);
-            relayBalance = await web3.eth.getBalance(relayServerAddress);
-            i++
-        }
-        assert.equal(0,relayBalance)
-        let afterOwnerBalance = await web3.eth.getBalance(relayOwner);
-        assert.equal(true,parseInt(afterOwnerBalance)  > parseInt(beforeOwnerBalance))
+    describe("relay balance management", async function () {
+        let relayServerAddress;
+        let beforeOwnerBalance;
+        it("should NOT send relay balance to owner after removed", async function () {
+            let response = await request(localhostOne+'/getaddr');
+            relayServerAddress = JSON.parse(response.body).RelayServerAddress;
+            beforeOwnerBalance = await web3.eth.getBalance(relayOwner);
+            let res = await rhub.removeRelayByOwner(relayServerAddress, {from:relayOwner});
+            assert.equal("RelayRemoved", res.logs[0].event);
+            assert.equal(relayServerAddress.toLowerCase(), res.logs[0].args.relay.toLowerCase());
+
+            let i = 0;
+            let relayBalance = await web3.eth.getBalance(relayServerAddress);
+            while (relayBalance != 0 && i < 20) {
+                await testutils.sleep(200);
+                relayBalance = await web3.eth.getBalance(relayServerAddress);
+                i++
+            }
+            let afterOwnerBalance = await web3.eth.getBalance(relayOwner);
+            assert.equal(false,parseInt(afterOwnerBalance)  > parseInt(beforeOwnerBalance))
```

Comment:
Huh? Either do assert.ok, our (better) asset.equal.
Not really sure why you test it this way: the relay don't do any operation in this period, so you should do assert equal (before, after)

Reply:
Yes I'm tired